src/main/scala/guesswho/GameCharacter.scala

```diff
@@ -1,3 +1,3 @@
 package guesswho
 
-case class GameCharacter(val name: String, val attributes: Map[Attribute, Any])
+case class GameCharacter(name: String, attributes: Map[Attribute, Any])
```

Comment:
what's the difference between having vals here or not

Reply:
When injecting with Play in a class you can use/not use the val X: Y too. Do you know what the difference in functionality here would be? (it seems to not make a difference for case classes)

Reply:
For a normal class, not putting val means the parameter is a constructor parameter but not a field, so it can't be accessed from outside the class